ios/KMMIC/Sources/Presentation/Modules/SurveyDetail/SurveyDetailScreen.swift

```diff
@@ -29,7 +30,15 @@ struct SurveyDetailScreen: View {
                 HStack {
                     Spacer()
                     PrimaryButton(R.string.localizable.surveyDetailScreenStartButtonTitle()) {
-                        // TODO: Begin survey
+                        guard let surveyDetail = viewModel.survey.detail else { return }
+                        let viewModel = SurveyQuestionViewModel(
+                            surveyDetail: surveyDetail,
+                            questionIndex: 0
+                        )
+                        navigator.show(
+                            screen: .surveyQuestion(viewModel: viewModel),
+                            by: .presentCover
+                        )
```

Comment:
let get the `SurveyQuestionViewModel` from viewModel.
```suggestion
                        let surveyQuestionViewModel = viewModel.surveyQuestionViewModel
                        navigator.show(
                            screen: .surveyQuestion(viewModel: surveyQuestionViewModel),
                            by: .presentCover
                        )
```

Reply:
@suho updated at [279d78c](https://github.com/markgravity/kmm-ic/pull/88/commits/279d78ce7c4f986afed3b97d8f4e79fab139fd9c)